packages/file-upload/src/FileUpload.tsx

```diff
@@ -103,28 +104,35 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 			maxFiles
 		);
 		const acceptedFilesSummary = getAcceptedFilesSummary(accept);
+		const styles = fileInputStyles({
+			disabled,
+			invalid: invalid || !!errorSummary,
+			valid,
+		});
 
 		return (
-			<Stack gap={0.5}>
-				<Field
-					label={label}
-					required={Boolean(required)}
-					hint={hint}
-					message={message || errorSummary}
-					invalid={invalid || !!errorSummary}
-					valid={valid}
-					id={id}
-				>
-					{(allyProps) => (
-						<div {...getRootProps()}>
+			<Field
+				label={label}
+				required={Boolean(required)}
+				hint={hint}
+				message={message || errorSummary}
+				invalid={invalid || !!errorSummary}
+				valid={valid}
+				id={id}
+			>
+				{(allyProps) => {
+					// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
@jordanoverbye I was trying to keep TS happy here, as 'color' is not supported by Stack. Maybe you can suggest a better approach?